### PR TITLE
RelatedPerson - Documentation

### PR DESCRIPTION
**Pull request by cerbs**
Documentation for the RelatedPerson resource

Note:

There are no terminology bindings or implementation notes since no non-required terminologies supported and no ignored modifying elements

Built from the same base as #84, so this includes changes to break apart the example json into individual files per resources

### CODE REVIEW

**Comment by daliboz** [comment]
Similar comment as patient: probably need to include binding for identifier type

**Comment by daliboz** [comment]
Otherwise +1

File: content/dstu2/related-person.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+    GET /RelatedPerson?:parameters
+
+### Parameters
+
+ Name         | Required?                                                    | Type          | Description
+--------------|--------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------
+ `_id`        | No, if populated all other parameters are ignored            | [`token`]     | The logical resource id associated with the resource.
+ `identifier` | Yes, if neither `_id` nor `patient` are set                  | [`token`]     | A RelatedPerson identifier.
+ `patient`    | Yes, if neither `_id` nor `identifier` are set               | [`reference`] | A reference to a patient related which the RelatedPerson is associated. Example: `14067892`
```
Comment (MaxPhilips) [inline content/dstu2/related-person.md]: Could trim out some of the empty space in this table to keep the style consistent with other resources :)

```
 Name         | Required?                                         | Type          | Description
--------------|---------------------------------------------------|-------------------------------------------------------------------------------------------------------------
 `_id`        | No, if populated all other parameters are ignored | [`token`]     | The logical resource id associated with the resource.
 `identifier` | Yes, if neither `_id` nor `patient` are set       | [`token`]     | A RelatedPerson identifier.
 `patient`    | Yes, if neither `_id` nor `identifier` are set    | [`reference`] | A reference to a patient related which the RelatedPerson is associated. Example: `14067892`
```

**Comment by MaxPhilips** [comment]
+1

**Comment by cerbs** [comment]
Rebasing

**Comment by cerbs** [comment]
Merged and squashed on commit ab04bcc

**Pull request closed without merging**